components/prize_pool/commons/prize_pool_placement_base.lua

```diff
@@ -174,7 +174,7 @@ end
 
 ---@param opponent BasePlacementOpponent
 ---@param prize string
----@return unknown
+---@return string|number|boolean?
```

Comment:
When is boolean a valid return?

Reply:
`qualified` input gets parsed to a bool

https://github.com/Liquipedia/Lua-Modules/blob/1ec56c9a878cb13c0fd3956b571815ebc3a32139/components/prize_pool/commons/prize_pool_base.lua#L243